src/components/TokenBottomSheet.tsx

```diff
@@ -259,48 +272,45 @@ function TokenBottomSheet<T extends TokenBalance>({
             />
           )
         }}
-        ListHeaderComponent={
-          <>
-            <Text style={[styles.title, titleStyle]}>{title}</Text>
-            {searchEnabled && (
-              <SearchInput
-                placeholder={t('tokenBottomSheet.searchAssets') ?? undefined}
-                value={searchTerm}
-                onChangeText={(text) => {
-                  setSearchTerm(text)
-                  sendAnalytics(text)
-                }}
-                style={styles.searchInput}
-                returnKeyType={'search'}
-                // disable autoCorrect and spellCheck since the search terms here
-                // are token names which autoCorrect would get in the way of. This
-                // combination also hides the keyboard suggestions bar from the top
-                // of the iOS keyboard, preserving screen real estate.
-                autoCorrect={false}
-                spellCheck={false}
-              />
-            )}
-            {filterChips.length > 0 && (
-              <FilterChipsCarousel
-                chips={filters}
-                onSelectChip={handleToggleFilterChip}
-                primaryColor={colors.successDark}
-                secondaryColor={colors.successLight}
-                style={styles.filterChipsCarouselContainer}
-                forwardedRef={filterChipsCarouselRef}
-              />
-            )}
-          </>
-        }
-        ListHeaderComponentStyle={styles.headerContainer}
-        stickyHeaderIndices={[0]}
+        ListHeaderComponent={<View style={{ height: headerHeight }} />}
         ListEmptyComponent={() => {
           if (searchEnabled || filterChips.length > 0) {
             return <NoResults searchTerm={searchTerm} activeFilters={activeFilters} />
           }
           return null
         }}
       />
+      <View style={styles.headerContainer} onLayout={handleMeasureHeader}>
```

Comment:
since the bottom sheet is only mounted once, the header is only measured once. if reusing the same bottom sheet with different headers, this isn't going to work (but we don't have a use case for that so i didn't worry much about it)

Reply:
JK, that's not how onLayout works. this will be fine because onLayout is called when the layout changes too